### PR TITLE
Add class to bindmount the ephemeral /mnt to /localscratch

### PR DESCRIPTION
**Pull request by matthewbenstead**
Add class to bindmount the ephemeral /mnt to /localscratch

### CODE REVIEW

File: manifests/site.pp [hunk 1]
```diff
@@ -22,6 +22,7 @@
     include profile::ssh::hostbased_auth::client
     include profile::ceph::client
     include profile::ccsudo::sudoers
+    include profile::localscratch::ephemeraldisk
```
Comment (cmd-ntrf) [inline manifests/site.pp]: `File['/localscratch']` is created by `profile::slurm::node`, so it is missing on login node.

Unless there is a need for /localscratch on login-node that I am not aware of, remove this line.

**Review by cmd-ntrf — CHANGES_REQUESTED** [review]
fix site.pp

**Comment by matthewbenstead** [comment]
> fix site.pp

Should be fixed now

File: manifests/site.pp (hunk 1 above)
Comment (matthewbenstead) [reply manifests/site.pp]: Removed from login node

**Review by cmd-ntrf — APPROVED** [review]

**Pull request merged**